woocommerce/woocommerce
# Add the wp wc com extension list command

### All Submissions:






### Changes proposed in this Pull Request:

This PR adds the command `wp wc com extension list` which displays a list of extensions purchased by the user who connected the site to WCCOM. The result in this command can be used in `wp wc extension install` command to find which extensions can be installed on the site by the user.

Closes # .

### How to test the changes in this Pull Request:

1. Make sure the site is connected to WCCOM via `<your test site root>wp-admin/admin.php?page=wc-addons&section=helper`
2. Run `wp wc com extension list`
3. This should display a list of extensions available for the site.
![Screenshot 2022-07-08 at 15 24 13](https://user-images.githubusercontent.com/1531403/178000645-5e54bc84-6520-41df-ae2a-ed52acf6120d.png)


### Other information:




### FOR PR REVIEWER ONLY:
